src/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.sol

```diff
@@ -303,6 +377,25 @@ contract BancorVirtualSupplyBondingCurveFundingManager is
         );
     }
 
+    /// @dev Calculates the static price for either selling or buying the issuance token,
+    /// based on the provided issuance token supply, collateral supply, and buy or sell reserve ratio.
+    /// Note: The reserve ratio specifies whether the sell or buy price is returned.
+    /// The formula used is: PPM * PPM * collateralSupply / (issuanceTokenSupply * reserveRatio).
+    /// The formula is based on Aragon's BatchedBancorMarketMaker, which can be found here:
+    /// https://github.com/AragonBlack/fundraising/blob/5ad1332955bab9d36cfad345ae92b7ad7dc0bdbe/apps/batched-bancor-market-maker/contracts/BatchedBancorMarketMaker.sol#L415
+    /// @param _issuanceTokenSupply The total supply of the issuance tokens.
+    /// @param _collateralSupply The total supply of the collateral held by the FundingManager.
+    /// @param _reserveRatio The reserve ratio, specified as either sell or buy reserve ratio.
+    /// @return uint The calculated static price for the specified operation, formatted in PPM
+    function _staticPricePPM(
```

Comment:
Just for my understanding:
What exatcly is the static Price?
Shouldnt this be linked in some form to the BancorFormular, as the price changes based on demand?

Reply:
static price is basically "where the needle is at" in the bonding curve. Any actual amount bought will move it up (so the final price will be different), but it kind of represents where the price starts moving from